Sketcher/SketchUnmap.FCMacro

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python
+#####################################
+# Copyright (c) openBrain 2019
+# Licensed under LGPL v2
+#
+# This macro will unmap a sketch from eg. a face and make its placement absolute in the body. It proposes 3 options (forcely the first if sketch not in a body) :
```

Comment:
Please mention FreeCAD in this this description like `This FreeCAD macro will unmap...`

Reply:
OK. Why not precise it on the readme of the repo ? ;)

Reply:
Yea, good idea. Needs to go there.